### PR TITLE
Game updater prototype

### PR DESCRIPTION
**Pull request by Elinvynia**
This PR is a prototype - currently it just displays the versions of client and server and doesn't do anything else. I kept the PR small to make it easier to review.

The server keeps it's own version (user cannot fake this) while there is also a client version (user can fake this but malformed packets don't get processed).

It also adds endpoints for the web to download the zip of the game. This will need more scripts in the future to run everything nicely from one place, I'll work on that!

Less relevant but still necessary changes:
No need for check.sh anymore thanks to workspaces.
Created a script for the database.

Future TODO:
Add version to connect/every packet.
Make getting the client version smoother, currently it opens the game for 0.1 sec.

### CODE REVIEW

File: enfaria-game/src/menu/mainmenu.gd [hunk 1]
```diff
@@ -7,6 +7,12 @@ onready var path = get_node("/root/constants").config_path
 var config = ConfigFile.new()
 
 func _ready():
+    var args = OS.get_cmdline_args()
+    if len(args) > 0:
+        if args[0] == "--v":
+            print(get_node("/root/env").get("VERSION"))
```
Comment (meesha7) [inline enfaria-game/src/menu/mainmenu.gd:13]: Isn't there a better way to parse the args? Like if the user provides `--v` as a second or third args

File: enfaria-web/src/release.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+use crate::prelude::*;
+
+pub fn routes(app: &mut Server<State>) {
+    app.at("release/windows").get(|_| async move {
+        let file = std::fs::read("release/windows/enfaria-game.zip")?;
```
Comment (meesha7) [inline enfaria-web/src/release.rs:5]: I know this is a prototype but the zip names should contain the version string

**Review by meesha7 — CHANGES_REQUESTED** [review]

File: enfaria-game/src/menu/mainmenu.gd (hunk 1 above)
Comment (Elinvynia) [reply enfaria-game/src/menu/mainmenu.gd:13]: We don't support other args at all, the game itself will be launched via the updater. so if the user touches this, the game should just quit.

File: enfaria-web/src/release.rs (hunk 2 above)
Comment (Elinvynia) [reply enfaria-web/src/release.rs:5]: This is actually intended, only the latest version should be available for download (keep in mind this is how "we" or someone else will distribute the game, and has less to do with development)

Comment (Elinvynia) [reply enfaria-web/src/release.rs:5]: (Also as a sidenote, the updater will always try to get the same version as the server, so unless someone modifies this repo in a way that breaks client-server compatibility without properly updating versions, that we can't prevent)

**Review by meesha7 — APPROVED** [review]

**Pull request merged**